content/docs/ui/sending-email/deliverability.md

```diff
@@ -46,7 +46,7 @@ Think of it this way: would you rather a recipient politely decline future email
 
  ### 	Who are you?
 
-Maximum company visibility helps as well. Placing your company name in the subject line of your emails and including your physical mailing address and phone number in your email footers helps mail providers recognize you as a legitimate company and sender of email. This also helps your recipients know that this message is indeed from you!
+Maximum company visibility helps as well. Placing your company name in the subject line of your emails and including your physical mailing address and phone number in your email footers helps mail providers recognize you as a legitimate company and sender of an email. This also helps your recipients know that this message is indeed from you!
```

Comment:
To avoid confusion, I would rephrase the second sentence to:

"Placing your company name in the subject line of your emails and including your physical mailing address and phone number in your email footers helps mail providers recognize you as a legitimate company and email sender."